plenum/server/config_req_handler.py

```diff
@@ -17,15 +19,19 @@ class ConfigReqHandler(LedgerRequestHandler):
     write_types = {TXN_AUTHOR_AGREEMENT, TXN_AUTHOR_AGREEMENT_AML}
     query_types = {GET_TXN_AUTHOR_AGREEMENT, GET_TXN_AUTHOR_AGREEMENT_AML}
 
```

Comment:
Please don't forget to extend static validation in `ClientOperationField` for a new read request.

Reply:
That's strange - I really forgot to extend static validation, however these read requests are still working - that could be a serious bug in static validation.

Reply:
This is expected since `OPERATION_SCHEMA_IS_STRICT = False`, so new/unknown fields are currently allowed. This was done to support new fields in messages without breaking changes in the protocol.